modules/services/agentless-scanning/main.tf

```diff
@@ -50,6 +50,31 @@ data "aws_iam_policy_document" "scanning" {
     ]
   }
 
+  statement {
+    sid = "AllowKMSEncryptDecrypt",
+
+    action = [
```

Comment:
Terraform Data block `aws_iam_policy_document` has argument `actions` instead of `action`.  TF will error out with this.

Reply:
Thanks for fixing this.